bazel/ngraph.BUILD

```diff
@@ -168,7 +119,7 @@ cc_library(
         "-fstack-protector-all",
         '-D SHARED_LIB_PREFIX=\\"lib\\"',
         '-D SHARED_LIB_SUFFIX=\\".so\\"',
-        '-D NGRAPH_VERSION=\\"v0.20.0-rc.0\\"',
+        '-D NGRAPH_VERSION=\\"v0.25.1-rc.0\\"',
```

Comment:
We should document that these ngraph versions need to change in this checklist: https://github.com/tensorflow/ngraph-bridge/wiki/Release-Checklist

Reply:
Already did.